docs/fr/docs/advanced/path-operation-advanced-configuration.md

```diff
@@ -1,99 +1,99 @@
+### Utilisation du nom *path operation function* comme operationId
 
-If you want to use your APIs' function names as `operationId`s, you can iterate over all of them and override each *path operation's* `operation_id` using their `APIRoute.name`.
+Si vous souhaitez utiliser les noms de fonction de vos API comme `operationId`, vous pouvez les parcourir tous et remplacer chaque `operation_id` de *path operation* en utilisant leur `APIRoute.name`.
 
-You should do it after adding all your *path operations*.
+Vous devriez le faire après avoir ajouté toutes vos *paramètres de chemin*.
 
 ```Python hl_lines="2  12-21  24"
 {!../../../docs_src/path_operation_advanced_configuration/tutorial002.py!}
 ```
 
 !!! tip
-    If you manually call `app.openapi()`, you should update the `operationId`s before that.
+    Si vous appelez manuellement `app.openapi()`, vous devez mettre à jour les `operationId` avant.
 
 !!! warning
-    If you do this, you have to make sure each one of your *path operation functions* has a unique name.
+    Pour faire cela, vous devez vous assurer que chacun de vos *chemin* ait un nom unique.
 
-    Even if they are in different modules (Python files).
+    Même s'ils se trouvent dans des modules différents (fichiers Python).
 
-## Exclude from OpenAPI
+## Exclusion d'OpenAPI
 
-To exclude a *path operation* from the generated OpenAPI schema (and thus, from the automatic documentation systems), use the parameter `include_in_schema` and set it to `False`:
+Pour exclure un *chemin* du schéma OpenAPI généré (et donc des systèmes de documentation automatiques), utilisez le paramètre `include_in_schema` et assignez-lui la valeur `False` :
 
 ```Python hl_lines="6"
 {!../../../docs_src/path_operation_advanced_configuration/tutorial003.py!}
 ```
 
-## Advanced description from docstring
+## Description avancée de docstring
 
-You can limit the lines used from the docstring of a *path operation function* for OpenAPI.
+Vous pouvez limiter le texte utilisé de la docstring d'une *fonction de chemin* qui sera affiché sur OpenAPI.
 
-Adding an `\f` (an escaped "form feed" character) causes **FastAPI** to truncate the output used for OpenAPI at this point.
+L'ajout d'un `\f` (un caractère d'échappement "form feed") va permettre à **FastAPI** de tronquer la sortie utilisée pour OpenAPI à ce stade.
 
-It won't show up in the documentation, but other tools (such as Sphinx) will be able to use the rest.
+Il n'apparaîtra pas dans la documentation, mais d'autres outils (tels que Sphinx) pourront utiliser le reste.
 
 ```Python hl_lines="19-29"
 {!../../../docs_src/path_operation_advanced_configuration/tutorial004.py!}
 ```
 
-## Additional Responses
+## Réponses supplémentaires
 
-You probably have seen how to declare the `response_model` and `status_code` for a *path operation*.
+Vous avez probablement vu comment déclarer le `response_model` et le `status_code` pour une *opération de chemin*.
```

Comment:
Small remark, starting from this line the `*path operation*`  is translated, but not before
I feel like we could align them, in a direction on in the other :v: